Add missing server settings tracking events

zaerl opened the pull request:
Related to https://github.com/Automattic/dotcom-forge/issues/9590

## Proposed Changes

Add these new tracks
* `calypso_hosting_configuration_reset_db_password`
* `calypso_hosting_restore_plugins_and_themes`
* `calypso_hosting_configuration_toggle_sftp_card`
* `calypso_hosting_configuration_toggle_ssh_card`
* ~~`calypso_site_settings_admin_interface_style`~~

## Why are these changes being made?
<!--
It's easy to see what a PR does but much harder to find out why it was made,
particularly when researching old changes in history. Record an explanation of
the motivation behind this change and how it will help.
-->

* See https://github.com/Automattic/dotcom-forge/issues/9529

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Visit `http://calypso.localhost:3000/hosting-config/_BLOG_`
* Click all the links and buttons found in this issue https://github.com/Automattic/dotcom-forge/issues/9590
* Ensure on Tracks Vigilante that all the tracks are generated

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

zaerl committed:
Add missing server settings tracking events

fditrapani commented:
Hey @zaerl, I see you've added a needs design tag to this PR, how can we help?

zaerl commented:
> Hey @zaerl, I see you've added a needs design tag to this PR, how can we help?

Oh sorry. Wrong tag, by bad.

fditrapani commented:
No problem!

wongasy commented:
@zaerl Can we resolve the merge conflicts? @wojtekn Can YOLO help review these changes?

ouikhuan commented:
Hi @zaerl, try to moving this forward, could we rebase the PR and resolve the conflict here? So we can ask @wojtekn to review this PR?

zaerl committed:
Fix: remove section

zaerl commented:
@ouikhuan I have rebased this. The section with `calypso_site_settings_admin_interface_style` track has been removed in the meantime, so I removed it as well.

wojtekn reviewed (approved):
Looks good!
